lib/widget/camera.dart

```diff
@@ -158,7 +158,7 @@ class CameraScreenState extends WidgetState<CameraScreen>
     setState(() {});
   }
 
-  void defineErrorText() {
+  void getMaxSelectedMessage() {
```

Comment:
you should never change state in a "getxxxx' function. Please change the name of the function to configureOptions